discovery-infra/test_infra/helper_classes/kube_helpers/installenv.py

```diff
@@ -157,12 +198,26 @@ def _attempt_to_get_status() -> dict:
             expected_exceptions=KeyError
         )
 
+    def get_iso_download_url(self):
```

Comment:
needs to remove this duplicated function from the e2e test itself

Reply:
ack